frontend/src/routes/_oh._index/route.tsx

```diff
@@ -140,6 +140,15 @@ function Home() {
           />
         </div>
       </div>
+      <button
+        className="absolute bottom-4 right-4 text-white px-4 py-2 rounded cursor-pointer border-1 border-neutral-600"
+        onClick={() => {
+          window.location.href = "/app";
```

Comment:
```suggestion
          navigate("/app");
```
Further reading: https://remix.run/docs/hi/main/hooks/use-navigate